satpy/tests/reader_tests/test_mviri_l1b_fiduceo_nc.py

```diff
@@ -403,6 +416,7 @@ def test_get_dataset(self, file_handler, name, calibration, resolution,
             xr.testing.assert_allclose(ds, expected)
             assert ds.dtype == expected.dtype
             assert ds.attrs == expected.attrs
+            assert True
```

Comment:
Debugging leftover?

Reply:
True - sorry for that, will be fixed shortly.